features/mbedtls/inc/mbedtls/config.h

```diff
@@ -26,21 +27,6 @@
  */
 
 #ifndef MBEDTLS_CONFIG_H
-
-#include "platform/inc/platform_mbed.h"
-
-/*
- * Only use features that do not require an entropy source when
- * DEVICE_ENTROPY_SOURCE is not defined in mbed OS.
- */
-#if !defined(MBEDTLS_ENTROPY_HARDWARE_ALT) && !defined(MBEDTLS_TEST_NULL_ENTROPY)
-#include "mbedtls/config-no-entropy.h"
-
-#if defined(MBEDTLS_USER_CONFIG_FILE)
-#include MBEDTLS_USER_CONFIG_FILE
-#endif
-
-#else
```

Comment:
Note that this is a major change to the config.h. Unless this PR includes changes on how to configure Mbed TLS in Mbed OS in another way this is going to break pretty much every application that uses Mbed TLS in Mbed OS. Could you please explain why these changes are here?

Reply:
This change is a result of the automatic import process. It seems like some kind of an auto merge error. I will make sure there are no unnecessary differences between this tree and the upstream one.